.github/actions/build/action.yaml

```diff
@@ -58,6 +65,24 @@ runs:
           OKD_VERSION_TAG=${{ inputs.okd-version-tag }} \
           RPM_OUTDIR=/mnt/rpms
 
+        # Prepare the RPMs for upload to GitHub Internal Storage
+        cd /mnt/rpms
+        sudo tar zcvf /mnt/microshift-rpms-$(uname -m).tgz .
+
+    - name: Detect CPU architecture
+      id: arch
+      uses: ./.github/actions/arch
+
+    - name: Upload MicroShift RPMs to GitHub Internal Storage
+      if:  inputs.action == 'build-rpms' || inputs.action == 'build-all'
+      uses: actions/upload-artifact@v4
+      with:
+        name: microshift-rpms-${{ steps.arch.outputs.arch }}
+        path: /mnt/microshift-rpms-${{ steps.arch.outputs.arch }}.tgz
+        if-no-files-found: error
+        compression-level: 0
+        overwrite: true
```

Comment:
I'd remove it. I don't think you can upload artifacts to COPR - you can upload or point it to .spec file, then it runs the builds itself.
Maybe we'll need some kind of API to trigger the builds as commits in this repo are disconnected from actual microshift changes, but from what I see in the COPR, we won't be uploading the builds.

Also, the `if` has outdated input name